drivers/include/net/netdev.h

```diff
@@ -54,6 +54,7 @@ extern "C" {
 #include <sys/uio.h>
 
 #include "net/netstats.h"
+#include "net/l2filter.h"
```

Comment:
This is not sloppiness, this is moving tangential discussions (that stall the PR's review) to a separate thread BTW ;-).